apps/www/pages/_app.tsx

```diff
@@ -64,8 +67,29 @@ const WebApp = ({
   } = pageProps
 
   const router = useRouter()
+  const [allowlistAccess, setAllowlistAccess] = useState(false)
```

Comment:
This is probably not the best place to have this useEffect. Should be in its own hook or incorporated somewhere else. I just put it here because this is where we decide to show the paynotes - or not.